src/middleware.ts

```diff
@@ -10,15 +10,18 @@ export function createMiddleware(
 ): ExpressMiddleware {
   return async (req, res, next) => {
     const serverOrigin = `${req.protocol}://${req.get('host')}`
-    const mockedRequest = parseIsomorphicRequest({
-      id: '',
-      method: req.method,
+
+    const mockedRequest = new MockedRequest(
       // Treat all relative URLs as the ones coming from the server.
-      url: new URL(req.url, serverOrigin),
-      headers: new Headers(req.headers as HeadersInit),
-      credentials: 'omit',
-      body: req.body,
-    })
+      new URL(req.url, serverOrigin),
+      {
+        id: '',
```

Comment:
I don't think `id` is required to create a `MockedRequest`. It will be [passed as `init`](https://github.com/mswjs/msw/blob/ec57fbf1a8642e08d25a759bf86ea7885e5d5de5/src/utils/request/MockedRequest.ts#L93) to the `IsomorphicRequest` which `MockedRequest` extends, and generated there.

Reply:
Thanks, removed it.